createmobilespec/createmobilespec.js

```diff
@@ -56,7 +56,8 @@ function popd(dir) {
 }
 
 function pluginAdd(pluginName, searchPath, extraFlags) {
-    var command = cli + ' plugin add ' + pluginName + ' --searchpath ' + searchPath;
+    var sp = SEARCH_PATHS.hasOwnProperty(pluginName) ? SEARCH_PATHS[pluginName] : searchPath;
```

Comment:
It's clearer to put this logic outside of the function, on [line 568] (https://github.com/apache/cordova-mobile-spec/pull/128/files#diff-ad19d116ad56f7b3ae8ee9fdc8733f69R568). That way pluginAdd is only concerned with adding a plugin from a given path, not with extra logic regarding the path.